Headers/DebugServer2/Utils/Bits.h

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright (c) 2014-present, Facebook, Inc.
+// All rights reserved.
+//
+// This source code is licensed under the University of Illinois/NCSA Open
+// Source License found in the LICENSE file in the root directory of this
+// source tree. An additional grant of patent rights can be found in the
+// PATENTS file in the same directory.
+//
+
+#ifndef __DebugServer2_Utils_Bits_h
+#define __DebugServer2_Utils_Bits_h
+
+#include "DebugServer2/Utils/CompilerSupport.h"
+
+namespace ds2 {
+namespace Utils {
+
+template <typename T> static inline int FFS(T number) {
+#if __has_builtin(__builtin_ffs)
+  return __builtin_ffs(number);
+#else
+  for (int i = 1; number != 0; i++, number >>= 1) {
+    if (number & 1) {
+      return i;
+    }
+  }
+  return 0;
+#endif
+}
+
+template <typename T> static inline int PopCount(T number) {
+#if __has_builtin(__builtin_popcount)
+  return __builtin_popcount(number);
+#else
```

Comment:
The intrinsics should not be impacted by the WINAPI FAMILY.  Seems odd that the intrinsic would be unavailable since it should map pretty easily to `clz`.